src/Mapbender/CoreBundle/Component/Element.php

```diff
@@ -57,6 +57,34 @@
     /** @var string[] Element tag translation subjects */
     protected static $title = "mb.core.element.class.title";
 
+    /**
+     * @var string[]
+     * Determines if "frontendTemplate", "formTemplate", "adminType" should be inherited from the parent class
+     * EVEN IF THEY ARE AUTOMAGICALLY CALCULATED. The three strings above are the valid / effective values.
+     * Also supported is "all".
+     *
+     * E.g. a SuperHTMLElement extends HTMLElement extends Element, with all inheritance left enabled (default)
+     * and no own implementations would return
+     *  getHtmlTemplatePath(false) => "MapbenderCoreBundle:Element:htmlelement.html.twig"
+     *  getHtmlTemplatePath(true)  => "MapbenderCoreBundle:ElementAdmin:htmlelement.html.twig"
+     *  (same:) getFormTemplate()  => "MapbenderCoreBundle:ElementAdmin:htmlelement.html.twig"
+     *  getType()                  => "Mapbender/CoreBundle/Element/Type/HTMLElementAdminType"
+     *
+     * With inheritance off (empty array or or other falsy value, it would instead return:
+     *  getHtmlTemplatePath(false) => "MapbenderCoreBundle:Element:super_htmlelement.html.twig"
+     *  getHtmlTemplatePath(true)  => "MapbenderCoreBundle:ElementAdmin:super_htmlelement.html.twig"
+     *  (same:) getFormTemplate()  => "MapbenderCoreBundle:ElementAdmin:super_htmlelement.html.twig"
+     *  getType()                  => "Mapbender/CoreBundle/Element/Type/SuperHTMLElementAdminType"
+     * I.e. with inheritance off, new templates and new admin type would be required by default.
+     *
+     * @todo: add widgetName to inheritables? Does that make sense?
```

Comment:
Yes. There is no element without a widget fronted. Each widget has a unique name this need to be associated with an element PHP class. Potentially: more than one element back end can have one front-end widget.